__tests__/api-test.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index.js");
+const app = require('../app.js');
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topics to the client", () => {
+    return request(app).get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toEqual(expect.any(Array));
+        expect(topics.length).toBe(3);
```

Comment:
Really nice testing and it's good to see that you remembered to check for length of the array too.